### ISSUE TITLE
[Feature] Introduce ARIA attributes to the website for accessibility

### ISSUE THREAD

**Issue by Murdock9803** [creativecommons/cc-resource-archive#319]
## Problem

The current code in the website lacks better accessibility features such as `ARIA` functionality - Accessibility Rich Internet Applications.

## Description

By introducing these attributes, it would be easier for everyone to understand the contents on the website. Also, it will be good for screen readers too. We can include attributes such as `role`, `aria-label`, `aria-hidden`, etc..

## Additional context

[More context to WAI-ARIA](https://www.w3.org/WAI/standards-guidelines/aria/)

## Implementation

- [x] I would be interested in implementing this feature.

**Comment by UtkershBasnet** [comment]
I'd like to work on this issue

**Comment by TimidRobot** [comment]
@UtkershBasnet 
Please see [Contribution Guidelines — Creative Commons Open Source](https://opensource.creativecommons.org/contributing-code/) for how we manage issues and PRs.

We generally don't assign issues prior to resolution. Instead of asking for issues to be assigned, do the work and submit a pull request (PR). Even if multiple people submit PRs for the same issue, multiple ideas and implementations strengthen the final product.
